src/components/views/read/cvForm/Education.tsx

```diff
@@ -129,17 +109,18 @@ const Education: FC<Props> = ({
 
   const handleChange = (cardId: string, key: string, value: string): void => {
     setEducationCards((prevCards) => {
-      const index = educationCards.findIndex((card) => card.id === cardId);
-      const updatedCards = [...prevCards];
-      updatedCards[index] = {
-        ...updatedCards[index],
-        [key]: value,
-      };
+      const updatedCards = prevCards.map((card) =>
+        card.id === cardId ? { ...card, [key]: value } : card,
+      );
       return updatedCards;
     });
   };
-
+  const handlePrev = (): void => {
+    prevPage();
```

Comment:
You don't save the values when going to the previous section ?

Reply:
Unfortunately yes, I know it needs to be fixed, so it should have values, and it's noticable when you go back to a section all of the fields will be reset and empty, even that you have the whole object with filled values.